SiemensIXBlazor/Components/Card/Card.razor.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Components;
+using SiemensIXBlazor.Enums;
+
+namespace SiemensIXBlazor.Components
+{
+    public partial class Card
+    {
+        [Parameter, EditorRequired]
+        public string Id { get; set; } = string.Empty;
```

Comment:
The card component doesn't got an event, so the id property isn't required.

Reply:
You are right, I removed the Id property.